cmd/minikube/cmd/status.go

```diff
@@ -35,13 +38,20 @@ import (
 )
 
 var statusFormat string
+var output string
+
+type KubeconfigStatus struct {
+	Correct bool
```

Comment:
What is `Correct` in relation to `KubeconfigStatus`? The naming here does not seem intuitive to me.

Reply:
It means that the kubeconfig is correctly configured. I can change this to `CorrectlyConfigured` ?